src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,51 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final int STORAGE_MAX_CAPACITY = 10;
+    private final Object[] keys;
```

Comment:
It's better to use generics
```suggestion
    private final K[] keys;
```

Reply:
I thought about it, but such code doesn't compile. Error description: java: generic array creation

![Screenshot 2024-04-16 092346](https://github.com/mate-academy/jv-storage/assets/44746714/fa9db472-384e-4d59-8475-08cd5adc2318)